src/client/screens/Feed/index.js

```diff
@@ -215,7 +217,7 @@ class FeedPage extends React.Component {
               errorMessage={this.state.errorMessage}
               onSubmit={this.addPost.bind(this)}
             />
-            {postsToRender}
+            {this.state.isLoading === false ? <p>{postsToRender}</p> : <Loading />}
```

Comment:
we should not put the posts into a `<p>` tag, p tag is for text, we have a lot of more complex elements. We should use `<div>` instead

Reply:
you also can add a class to the `<div>`, when it is loading, you can use a `::before` or `::after` pseudo element, to add some overlay and the loading animation. In this case you don't need to hide the rendered posts